src/components/NoOrganisationFoundComponent.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import agreementIcon from "../images/agreement.svg";
+
+
+const NoOrganisationFoundComponent = () => {
```

Comment:
Should the name be like: NoContractFoundComponent instead?

Reply:
Done, nice catch !